src/kssh/ssh.go

```diff
@@ -58,7 +61,6 @@ func CreateDefaultUserConfigFile() error {
 	if err != nil {
 		return err
 	}
-	fmt.Printf("Using default ssh user %s\n", user)
```

Comment:
debug log?

Reply:
Yup. But I just moved the log statement to kssh.go since it felt more correct for it to be there when the `-F configfile` arguments were added.